share/templates.d/99-generic/runtime-install.tmpl

```diff
@@ -30,11 +30,15 @@ installpkg glibc-all-langpacks
 
 ## arch-specific packages (bootloaders etc.)
 %if basearch == "aarch64":
-    installpkg efibootmgr grub2-efi-aa64-cdboot grubby shim-aa64
```

Comment:
Are you sure grubby doesn't need to be here? Have you tested this template?

Reply:
grubby is explicitly there for all architectures on line 16 above that